.github/mergify.yml

```diff
@@ -0,0 +1,83 @@
+---
+defaults:
+  actions:
+    queue:
+      name: default
+      method: rebase
+      update_method: rebase
+
+
+# each test should be listed separately, do not use regular expressions:
+# https://docs.mergify.io/conditions.html#validating-all-status-check
+# TODO: Use mergify's recently added 'shared configuration support'
+# to dedup some of the check-x=y repetition in the future.
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=check
+      - check-success=build-server
+      - check-success=build-ad-server
+      - check-success=build-nightly-server
+      - check-success=build-nightly-adserver
+      - check-success=build-client
+      - check-success=build-toolbox
+      - check-success=test-ad--server-kubernetes
+      - check-success=test-nightly-ad-server-kubernetes
+      - check-success=dpulls
+
+
+pull_request_rules:
+  # Clearing approvals after content changes
+  - name: Remove outdated approvals
+    conditions:
+      - base=master
+    actions:
+      dismiss_reviews:
+        approved: true
+        changes_requested: false
+  # Perform automatic merge on conditions
+  - name: Automatic merge on approval
+    conditions:
+      - check-success=check
+      - check-success=build-server
+      - check-success=build-ad-server
+      - check-success=build-nightly-server
+      - check-success=build-nightly-adserver
+      - check-success=build-client
+      - check-success=build-toolbox
+      - check-success=test-ad--server-kubernetes
```

Comment:
See above.

Reply:
> See above.



> See above.


I think I fixed the double dashes yesterday ...